projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/qualifying-questions/qualifying-question-options/qualifying-question-options.component.html

```diff
@@ -14,16 +14,21 @@ <h1 class="govuk-fieldset__heading">
       </ng-container>
 
       <div class="govuk-radios" data-module="govuk-radios">
-        <div *ngFor="let qualifyingQuestion of qualifyingQuestions$ | async" class="govuk-radios__item">
-          <input class="govuk-radios__input"
-                 id="{{qualifyingQuestion.name}}"
-                 name="qualifyingQuestionOption"
-                 type="radio"
-                 [formControl]="qualifyingQuestionsControl"
-                 [value]="qualifyingQuestion">
-          <label class="govuk-label govuk-radios__label"
-                 [for]="qualifyingQuestion.name">{{ qualifyingQuestion.name | rpxTranslate }}</label>
-        </div>
+        <ng-container *ngFor="let qualifyingQuestion of qualifyingQuestions$ | async; let isLast = last">
+          <ng-container *ngIf="isLast">
```

Comment:
@MateiHMCTS there is no hr line based on the Figma I referred to https://www.figma.com/proto/uGkmgQzZWmHJYq7w6Y8ACa/Query-Management-Prototype-Latest-May-2023?page-id=783%3A72991&type=design&node-id=1061-88834&viewport=724%2C374%2C0.07&scaling=min-zoom&starting-point-node-id=1061%3A88834

Let us check with Tommy regarding this.